autotst/job/job.py

```diff
@@ -323,108 +376,220 @@ def check_isomorphic(conformer):
+                    label), shell=True)
+
+        # wait unitl the job is done
+        while not self.check_complete(label):
+            time.sleep(15)
+
+        # If the log file exits, check to see if it terminated normally
+        if os.path.exists(file_path + ".log"):
+            if orca_calc.check_NormalTermination(file_path + ".log"):
+                logging.info("The FOD calculation completed! The FOD log file is {}".format(
+                    file_path + ".log"))
+                copyfile(
+                    file_path + ".log", os.path.join(dir, label + ".log"))
+                return True
+            else:
+                logging.info("It appears the FOD job did not terminate normally! The FOD log file is {}".format(
+                    file_path + ".log"))
+                return False
+        else:
+            logging.info("It appears the FOD orca job never ran")
+            return False
+
+    def calculate_species(self, calculate_fod=False):
         """
         Calculates the energy and harmonic frequencies of the lowest energy conformer of a species:
         1) Systematically generates low energy conformers for a given species with an ASE calculator.
         2) Optimizes each low energy conformer with provided Gaussian AutoTST calculator.
         3) Saves the gaussian optimization and frequency analysis log file for the lowest energy conformer of the species.
         """
 
-        logging.info("Calculating geometries for {}".format(species))
+        if self.species is not None:
+            for species in self.species:
+                assert isinstance(species,Species),"species {} provided was not an AutoTST species object".format(species)
+        else:
+            raise TypeError("Species cannot be None, please specify an AutoTST species to calculate")
+        
+        calculation_status = dict()
+        for species in self.species:
+            logging.info("Calculating geometries for {}".format(species))
 
-        if self.conformer_calculator:
-            species.generate_conformers(ase_calculator=self.conformer_calculator)
+            if self.conformer_calculator:
+                species.generate_conformers(ase_calculator=self.conformer_calculator)
 
-        currently_running = []
-        processes = {}
-        for smiles, conformers in list(species.conformers.items()):
+            currently_running = []
+            processes = {}
+            for smiles, conformers in list(species.conformers.items()):
 
-            for conformer in conformers:
+                for conformer in conformers:
 
-                process = Process(target=self.calculate_conformer, args=(
-                    conformer,))
-                processes[process.name] = process
+                    process = Process(target=self.calculate_conformer, args=(
+                        conformer,))
+                    processes[process.name] = process
```

Comment:
This will be slow, but it'll get the job done... 

What will happen: 
- For each species
  - Generate conformers (which takes time)
  - Optimize conformers (which takes more time)
  - Calculate FOD for the lowest energy conformer

BUT, this is run one species at a time... so this will take a long while. Parallelize this plz